auth/libraries/pip-auth/authlib/config.py

```diff
@@ -16,18 +16,16 @@ def __init__(self, debug_fn: Any = print) -> None:
 
     def __load_keys(self) -> None:
         self.debug_fn("Environment.__load_keys")
-        default = AUTH_DEV_KEY if self.is_dev_mode() else None
-        self.debug_fn(f"Is dev-mode {self.is_dev_mode()} with default key {default}")
-        auth_token_key = os.getenv("AUTH_TOKEN_KEY", default)
-        auth_cookie_key = os.getenv("AUTH_COOKIE_KEY", default)
-        if not auth_token_key:
-            raise KeyException("No AUTH_TOKEN_KEY defined.")
-
-        if not auth_cookie_key:
-            raise KeyException("No AUTH_COOKIE_KEY defined.")
-
-        self.auth_token_key = auth_token_key
-        self.auth_cookie_key = auth_cookie_key
+        if (self.is_dev_mode()):
+            self.auth_token_key = AUTH_DEV_KEY
+            self.auth_cookie_key = AUTH_DEV_KEY
+        else:
+            self.auth_token_key = self.get_file_from_path("/run/secrets/auth_token_key")
+            if not auth_token_key:
+                raise KeyException("No AUTH_TOKEN_KEY defined.")
```

Comment:
Maybe include the path in the error message, so it is clear, that it is not a missing env variable.